src/cloud/benchflow/sender/sender.go

```diff
@@ -196,23 +218,43 @@ func consumeFromTopic(t TransformerSetting) {
 
 func submitAnalyser(script string, trialID string, minioKey string) {
 	var args []string
+	args = append(args, "--jars", sparkHome+"/pyspark-cassandra-assembly-0.2.7.jar")
+	args = append(args, "--driver-class-path", sparkHome+"/pyspark-cassandra-assembly-0.2.7.jar")
+	args = append(args, "--py-files", sparkHome+"/pyspark-cassandra-assembly-0.2.7.jar")
 	args = append(args, "--master", "local[*]")
-	args = append(args, "--packages", "TargetHolding:pyspark-cassandra:0.2.2")
+	//args = append(args, "--packages", "TargetHolding:pyspark-cassandra:0.2.2")
 	args = append(args, script)
 	args = append(args, "local[*]")
 	args = append(args, os.Getenv("CASSANDRA_IP"))
 	args = append(args, trialID)
 	args = append(args, minioKey)
 	fmt.Println(args)
+	retries := 0
 	cmd := exec.Command(sparkHome+"/bin/spark-submit", args...)
-	cmd.Stdout = os.Stdout
-	cmd.Stderr = os.Stderr
-	err := cmd.Start()
-	cmd.Wait()
-	if err != nil {
-		panic(err)
+	for true {
```

Comment:
The same as the previous `for true`. Moreover, since you have this code twice, I would use a function.